src/enums/document-types.ts

```diff
@@ -0,0 +1,642 @@
+export enum DocumentClasses {
+  CORP = 'CORP', // Corporations
+  COOP = 'COOP', // Cooperatives
+  FIRM = 'FIRM', // Firms
+  LP_LLP = 'LP_LLP', // Limited Partnership/Limited Liability Partnership
+  NR = 'NR', // Name Requests
+  SOCIETY = 'SOCIETY', // Societies
+  OTHER = 'OTHER', // Other
+  MHR = 'MHR', // Manufactured Home Registry
+  PPR = 'PPR', // Personal Property Registry
+}
+
+export enum DocumentTypes {
+  DIRECTOR_AFFIDAVIT = 'DIRECTOR_AFFIDAVIT',
+  CORP_AFFIDAVIT = 'CORP_AFFIDAVIT',
+  COOP_MEMORANDUM = 'COOP_MEMORANDUM', // Memorandum
+  COOP_RULES = 'COOP_RULES', // Rules
+  COU = 'COU', // Court Order
+  COOP_MISC = 'COOP_MISC', // Correction Filing
+  COFI = 'COFI', // Cooperatives Miscellaneous Documents
```

Comment:
Please alphabetize :)

Reply:
(this whole list)

Reply:
(within each section is OK)

You can also add whitespace between sections to make it easy to read.